crates/libs/result/Cargo.toml

```diff
@@ -21,9 +21,6 @@ workspace = true
 default-target = "x86_64-pc-windows-msvc"
 targets = []
 
-[dependencies]
-static_assertions = "1.0"
```

Comment:
This has just been moved to the `test_result` crate to avoid a production dependency.